ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/RequestHandlerTest.java

```diff
@@ -235,10 +252,65 @@ public void shouldOnlyReturnLastInRunScript() {
     assertThat(result, contains(entity2));
   }
 
```

Comment:
We should add a test that verifies that the changed properties are retained across statements.

Reply:
We have a test for this in `KsqlResourceTest`:
```java
  @Test
  public void shouldSetProperty() {
    // Given:
    final String csas = "CREATE STREAM " + streamName + " AS SELECT * FROM test_stream;";

    // When:
    final List<CommandStatusEntity> results = makeMultipleRequest(
        "SET '" + ConsumerConfig.AUTO_OFFSET_RESET_CONFIG + "' = 'earliest';\n"
            + csas,
        CommandStatusEntity.class);

    // Then:
    verify(commandStore).enqueueCommand(
        argThat(is(configured(
            preparedStatementText(csas),
            ImmutableMap.of(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest"),
            ksqlConfig))));

    assertThat(results, hasSize(1));
    assertThat(results.get(0).getStatementText(), is(csas));
  }
```